cmake/cores/Cortex-M4F.cmake

```diff
@@ -38,10 +40,10 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} \
     ${ARM_FLAGS} \
 ")
 set(CMAKE_ASM_FLAGS "${CMAKE_ASM_FLAGS} \
-    ${ARM_FLAGS} \
+    --cpu=Cortex-M4 \
 ")
 set(CMAKE_CXX_LINK_FLAGS "${CMAKE_CXX_LINK_FLAGS} \
-    ${ARM_FLAGS} \
+    --cpu=Cortex-M4 \
```

Comment:
are these changes relevant only to Cortex-M4F (the rest of cores are good?) ?

Reply:
@0xc0170 The `-mfpu` is relevant for cores with `F` appended to its name. But assembler and linker flags needs to be updated for all other cores.
Can we merge this PR as it works with K64F which is the chosen platform?

I'll create another PR with changes needed in other core cmake script.

Reply:
yes, i was just checking if we still got a work to do